Documentation/ApiOverview/CachingFramework/Developer/Index.rst

```diff
@@ -60,20 +61,17 @@ should hint an integrator about specific caching needs or setups in this case.
        $GLOBALS['TYPO3_CONF_VARS']['SYS']['caching']['cacheConfigurations']['myext_mycache']['backend'] = \TYPO3\CMS\Core\Cache\Backend\TransientMemoryBackend::class;
    }
 
-To get an instance of a cache, :code:`GeneralUtility::makeInstance(\TYPO3\CMS\Core\Cache\CacheManager::class)->getCache('myext_mycache')`
-can be used. However, you should prefer the dependency injection method (see below) to retrieve cache frontends whenever possible. The cache manager will return the fully initialized cache instance::
 
-   $myCacheInstance = GeneralUtility::makeInstance(\TYPO3\CMS\Core\Cache\CacheManager::class)->getCache('myext_mycache');
+.. _caching-developer-example:
+.. _caching-developer-access:
```

Comment:
_caching-developer-access is a strange key for Headline "Using the Chache"

Reply:
It might be the labels got moved around. Usually we try not to remove labels because links with :ref: will break if still pointing to it. But, I'll have a look ...